hack/ginkgo-version.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+gingko_vendor="github.com/onsi/ginkgo/v2"
```

Comment:
Let's just replace the ginkgo CLI with `go test` so we remove infra.

Reply:
ack, removed